@re-/model/demos/generateCode/mapTSfiles.ts

```diff
@@ -1,56 +1,133 @@
-import { fromHere } from "@re-/node"
-import { Project, Statement, ts } from "ts-morph"
+import { Project, SourceFile } from "ts-morph"
+import { Stack } from "../stack.js"
+import { addBlockToMaps, getComment } from "./helpers.js"
 import { regex } from "./regex.js"
 
-const { tagMatch, blockCommentMatch, commentMatch } = regex
+const {
+    blockEndGrab,
+    blockOrStatementGrab,
+    nameGrab,
+    includeGrab,
+    excludeGrab,
+    fileGrab,
+    statementGrab,
+    blockGrab
+} = regex
 
-const getComment = (text: string, range: any) =>
-    text.slice(range.getPos(), range.getEnd())
+const codeDestinations = ["stackBlitz", "tests", "docs"]
```

Comment:
Should be `CODE_DESTINATIONS` based on the way the rest of the constants are named.